lms/djangoapps/support/static/support/jsx/entitlements/data/api/endpoints.js

```diff
@@ -0,0 +1,3 @@
+export {
+  entitlementList: '/support/entitlement_list',
```

Comment:
This seems to be a constant at its root as well. You might want to make this all caps as well to match the code style.

Reply:
I am researching this within out code style guides.  This is the standard I have worked with in the past, but I am not sure where this fits in the edX style guide.

I have always found it to be more clear and helpful in the past.

Reply:
I was modeling this off of [studio front end endpoints](https://github.com/edx/studio-frontend/blob/master/src/data/api/endpoints.js). I'm not sure SCREAMING_SNAKE_CASE is out standard for constants in front end. @edx/fedx-team do we have a decision/opinion on that?

Reply:
I am fine with what ever the style guide says :D.  I have seen it different depending on the company.

Reply:
I think studio-frontend might have a mistake there. The JS convention is to use the SCREAMING_SNAKE_CASE :snake: for constants.

Reply:
@Mjloturco and @staubina I think we default to the `airbnb/javascript` style guide. Currently, it looks like [the rule is that constants should be defined as camelCase](https://github.com/airbnb/javascript#naming--camelCase).

However, there is [this PR that might add SCREAMING_SNAKE_CASE](https://github.com/airbnb/javascript/pull/255).

Reply:
Got a couple of other opinions.  You can disregard these recommendations. This is more a thing in Python and other languages.

Reply:
Oh, guess I'm wrong. I thought our style guide was different. Redux uses SCREAMING_SNAKE_CASE for action name constants so I thought it was the same as that.

Reply:
Cool, leaving it as is.